identityClient/api.go

```diff
@@ -923,6 +924,17 @@ type AccessRequestResponse struct {
 	RealmName             string               `json:"realm_name"`
 }
 
+// ApproveAccessRequestsRequest specifies a list of AccessRequests to approve
+type ApproveAccessRequestsRequest struct {
+	AccessRequestIDs []int64 `json:"access_request_ids"`
```

Comment:
Post demo we may want to change this to accept an object, so we can include notes?

Reply:
agreed

Reply:
comments now part of request payload